assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -166,18 +243,55 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 					</BaseControl>
 				</PanelBody>
 			</InspectorControls>
+			{ colorGradientSettings.hasColorsOrGradients && (
+				<InspectorControls group="color">
```

Comment:
I'm not sure `group` is an expected prop? Shouldn't it be `key`? Maybe look into that.